File: python_MVC/service/serviceImpl/service_popDao_pushDaoImpl.py
# serviceImpl/service_popDao_pushDaoImpl.py

from service.ServicePopDaoPushDao import ServicePopDaoPushDao
from dao.daoImpl.PushImpl import PushImpl
from dao.daoImpl.PopImpl import PopImpl
from entity.Filed import Filed

class ServicePopDaoPushDaoImpl(ServicePopDaoPushDao):
    def __init__(self):
        self.pusher = PushImpl()
        self.popper = PopImpl()

    def push_data(self, stack: Filed, data: int):
        self.pusher.push(stack, data)

    def pop_data(self, stack: Filed) -> int:
        return self.popper.pop(stack)

File: java_MVC/Service/ServiceImpl/Service_popDao_pushDaoImpl.java
package Service.ServiceImpl;

import dao.Push;
import dao.Pop;
import dao.daoImpl.PushImpl;
import dao.daoImpl.PopImpl;
import entity.Filed;
import Service.Service_popDao_pushDao;

public class Service_popDao_pushDaoImpl implements Service_popDao_pushDao {
    private Push pushDao = new PushImpl();
    private Pop popDao = new PopImpl();

    @Override
    public void pushData(Filed stack, int data) {
        pushDao.push(stack, data);
    }

    @Override
    public int popData(Filed stack) {
        return popDao.pop(stack);
    }
}
